onnxruntime/test/python/onnxruntime_test_python_cudagraph.py

```diff
@@ -50,7 +94,8 @@ def testRunModelWithCudaGraph(self):
             io_binding.bind_ortvalue_input("X", x_ortvalue)
             io_binding.bind_ortvalue_output("Y", y_ortvalue)
 
-            # One regular run for the necessary memory allocation and cuda graph capturing
+            # Two regular runs for the necessary memory allocation and cuda graph capturing
```

Comment:
There is a test here too that needs correpsonding updates - https://github.com/microsoft/onnxruntime/blob/8b09702b88a2ed2258c5e5a5f63a5b539ac93f43/onnxruntime/test/shared_lib/test_inference.cc#L1685